optimum/onnxruntime/modeling_ort.py

```diff
@@ -574,6 +574,58 @@ def _from_transformers(
             provider_options=provider_options,
         )
 
+    @classmethod
+    def _from_timm(
```

Comment:
Need Suggestion: 
The method is exact same as `from_transformers`.  Do we wanna keep them separate for easy reading / just use a single method.